controllers/Users/userController.js

```diff
@@ -0,0 +1,38 @@
+const User = require('../../lib/modelManagers/usermodel');
+
+const pagination = require('../../lib/utils/pagination/paginationHelper');
+
+/**
+ *
+ *
+ * @class Products
+ */
+class UsersController {
+  /**
+   * @static
+   * @returns { object } undefined
+   * @param {*} req
+   * @param {*} res
+   */
+  static async listUsers(req, res) {
+    const { limit, offset } = pagination(req.query);
+    try {
+      const users = await User.listAllUsers(limit, offset);
+      if (users) {
+        res.status(200).send({
+          msg: 'users retrieved succesfully ',
+          users,
+        });
+      } else {
+        res.status(400).send({
+          msg: 'error retrieving users',
+        });
+      }
+    } catch (error) {
+      throw error;
```

Comment:
` throw error;` are you catching this error somewhere?

Reply:
i have fixed this, i specified a response when an error is thrown